tests/math/math_utils_test.py

```diff
@@ -0,0 +1,27 @@
+# Copyright OTT-JAX
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import jax
+import jax.numpy as jnp
+from ott.math import utils as mu
+
+
+class TestNorm:
+
```

Comment:
Please add a new parameterized that checks the new `mu.norm` is the same for non-0 input as `jnp.linalg.norm`.

Reply:
This might be helpful (to compare with finite difference as well; we should use it in other places): `jax.test_util.check_grads`.

Reply:
thanks! there was indeed a bug (inverted args in `jnp.where`)